src/odemis/driver/xt_client.py

```diff
@@ -1638,6 +1677,19 @@ def _get_acq_msg(self, **kwargs):
             logging.warning("Acq received unexpected message %s", msg)
         return msg
 
+    def _set_scanner(self, scan_mode):
+        """
+        Setter for changing the current active scanner type. The correct scanner object is also updated in ._scanner.
+        :param scan_mode (string): contains mode, can be either 'Scanner' or 'FibScanner'
```

Comment:
Please update the docstring. I'd recommend renaming "scan_mode" -> "name" or "scanner_name".